sdk/python/distillation/benchmarking/evaluating_distill_models.ipynb

```diff
@@ -0,0 +1,217 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "source": [
+    "# Evaluating Distilled Models with AzureML\n",
+    "\n",
+    "In this notebook, you will learn how to run evaluations of opensource distilled model using the AzureML SDK. Along with this notebook, we've included a preconfigured set of 5 tasks using well-known public datasets.\n",
+    "\n",
+    "*Disclaimer: This notebook has been tested against MaaS endpoints for Llama 3.1. Other deployments or model versions are not guaranteed to work with the evaluation pipelines distributed with this notebook.*  \n",
+    "\n",
+    "## Prerequistes\n",
+    "- An Azure account with an active subscription - [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F)\n",
+    "- An Azure ML workspace - [Configure workspace](../../configuration.ipynb)\n",
+    "- Installed Azure Machine Learning Python SDK v2 - [install instructions](../../../README.md) - check the getting started section\n",
+    "- A python environment with [mlflow](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-use-mlflow-configure-tracking?view=azureml-api-2&tabs=python%2Cmlflow) for retrieving eval metrics\n",
+    "- Distilled model endpoint url and connection name for workspace connection\n",
+    "\n",
+    "## Supported tasks\n",
+    "- Summarization\n",
+    "- Math\n",
+    "- NLI\n",
+    "- NLU-QA\n",
+    "- Conversation\n",
+    "\n",
+    "Note that evaluation pipelines automatically download relevant datasets from public sources.\n",
+    "\n",
+    "You can also set the sample ratio, the fraction of the selected dataset to run for the eval.\n",
+    "\n",
+    "**Warning**: Many datasets contain thousands of examples which can lead to high endpoint usage costs. We advise starting with a small sample ratio (e.g., 1%) to verify the pipeline and then increasing the ratio if desired. Note that benchmark metrics obtained with small sample ratios may not be comparable between different models. Please use sample_ratio=1 for model comparisons.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# AzureML settings - please fill in your values\n",
+    "subscription_id = \"<Azure subscription ID>\"\n",
+    "resource_group = \"<Resource group>\"\n",
+    "workspace_name = \"<Workspace name\"\n",
+    "experiment_name = \"<Experiment name>\"\n",
+    "\n",
+    "# Eval to run - you can change this to any of the 5 supported task names\n",
+    "# Supported tasks: summarization, math, nli, nlu_qa, conversation\n",
+    "task_name = \"summarization\"\n",
+    "\n",
+    "# Supported datasets: hellaswag, openbookqa, snli, gsm8k, dialogsum\n",
```

Comment:
We should probably clarify in the comment that each dataset is for each task and they dont mix and match.

> \# Supported datasets:
> \# conversation: hellaswag 
> \# nlu_qa: openbookqa
> \# nli: snli
> \# math: gsm8k
> \# summarization: dialogsum